docs/packages/pkg/OpenCV.rst

```diff
@@ -1,6 +1,8 @@
 .. spelling::
 
     OpenCV
+    GStreamer
+    backend
```

Comment:
`backend` can be removed from here. In text `back-end` should be used:
* https://en.oxforddictionaries.com/definition/us/back_end